packages/ncurses/build.sh

```diff
@@ -61,6 +61,7 @@ termux_step_post_make_install() {
 		for file in lib${lib}w.so*; do
 			ln -s $file ${file/w./.}
 		done
+		rm -f lib${lib}.a
```

Comment:
Without this, the symbolic link below fails because it tries to link a file that's already there from the default ncurses install.

Reply:
You could also use `ln -sf $file ${file/w./.}` to force overwrite already existing symlinks, saves a line of code.

Reply:
Right now ncurses considered to be unsafe due to these `rm` operations. I guess we should go with `ln -sf` here, this will cause links to be correct and update timestamp, so everything should be okay just as with `rm`'s.

Reply:
Should be fixed with new way of creating symlinks.